### PR TITLE
Add Sequencer and RPC Metrics

### PR DESCRIPTION
**Pull request by doutv**
1. Collect more Prometheus metrics
2. Create XLayer Grafana dashboard
3. Run prometheus and grafana in docker compose test

### CODE REVIEW

File: zk/txpool/pool_zk.go [hunk 1]
```diff
@@ -114,9 +114,9 @@ func (p *TxPool) onSenderStateChange(senderID uint64, senderNonce uint64, sender
 					newGpBig.Set(dGp)
 					if claim {
 						newGpBig = newGpBig.Mul(newGpBig, big.NewInt(int64(p.xlayerCfg.GasPriceMultiple)))
-						log.Info(fmt.Sprintf("Free tx: type claim. dGp:%v, factor:%d, newGp:%d", dGp, p.xlayerCfg.GasPriceMultiple, newGpBig))
+						log.Debug(fmt.Sprintf("Free tx: type claim. dGp:%v, factor:%d, newGp:%d", dGp, p.xlayerCfg.GasPriceMultiple, newGpBig))
 					} else {
-						log.Info(fmt.Sprintf("Free tx: type newAddr. nonce:%d, dGp:%v, newGp:%d", mt.Tx.Nonce, dGp, newGpBig))
```
Comment (doutv) [inline zk/txpool/pool_zk.go:119]: Will we use these logs? I set log level to Debug because they are too much

**Comment by doutv** [comment]
close https://github.com/okx/xlayer-erigon/issues/174

**Review by zjg555543 — APPROVED** [review]

**Comment by zjg555543** [comment]
LGTM

File: zk/txpool/pool_zk.go (hunk 1 above)
Comment (doutv) [reply zk/txpool/pool_zk.go:119]: That's okay, we should keep debug log for the gp.

**Pull request merged**